### PR TITLE
Expose new custom components env vars to csv-generator and manifest-templator

### PR DESCRIPTION
**Pull request by iholder-redhat**
**What this PR does / why we need it**:
This is a follow-up for https://github.com/kubevirt/kubevirt/pull/8673. This PR aims to expose the new env vars introduced in the previous PR to csv-generator and manifest-templator.

**Release note**:

```release-note
Expose new custom components env vars to csv-generator and manifest-templator
```

### CODE REVIEW

**Comment by iholder-redhat** [comment]
/cc @jean-edouard @enp0s3 @fossedihelm

Can you please pay special attention to https://github.com/kubevirt/kubevirt/pull/8792/commits/25a82b6a703ddb513392b9da6a37511543314aa0?
Some context: currently, for some reason, it is allowed to provide either all of the custom [sha env vars](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-operator/util/config.go#L53) or none of them. This is a very old code and I guess this restriction isn't necessary. Am I missing anything?

**Review by jean-edouard — COMMENTED** [review]
/approve

**Review by fossedihelm — APPROVED** [review]
Thanks

File: tools/manifest-templator/manifest-templator.go [hunk 1]
```diff
@@ -41,6 +41,11 @@ import (
 	"kubevirt.io/kubevirt/tools/util"
 )
 
+const (
+	customImageExample   = "Examples: some.registry.com@sha256:abcdefghijklmnop, other.registry.com:tag1"
```
Comment (orenc1) [inline tools/manifest-templator/manifest-templator.go:45]: `abcdefghijklmnop` is not a valid example for sha digest as it should include only hexadecimal characters (0-9,a-f) :)

**Pull request merged**